data/evidence/status.go

```diff
@@ -0,0 +1,108 @@
+package evidence
+
+import (
+	"fmt"
+
+	"github.com/Oneledger/protocol/data/keys"
+	"github.com/Oneledger/protocol/serialize"
+	"github.com/Oneledger/protocol/storage"
+	"github.com/pkg/errors"
+)
+
+type ValidatorStatus struct {
```

Comment:
I'm not sure if we need to keep and maintain this extra active status using 100+ lines of code, because we already have this status in validator store, the lastActive field, and these are maintained by tendermint.